### ISSUE TITLE
Allow to call Display.setRescalingAtRuntime() after shells have been created

### ISSUE THREAD

**Issue by HeikoKlare** [vi-eclipse/Eclipse-Platform#115]
Calling `Display.setRescalingAtRuntime()` is currently only allowed when no shell has been instantiated yet, as otherwise the behavior is not determined.

The method implementation shall be extended so that it can also be called after shells have been created on the display.
- [ ] When activating rescaling via `Display.setRescalingAtRuntime(true), the shells shall be updated according to their current monitor's scaling.
- [ ] When deactivating rescaling via `Display.setRescalingAtRuntime(false)`, the current scale value shall be fixed, such that the shell behaves as if it was initialized with rescaling at runtime deactivated with the primary monitor having the current scale value of the shell.

**Comment by akoch-yatta** [comment]
After the investigation regarding #118 I don't know if this is possible, I would event assume it is not possible. Everything that is already created has inherited the DPI Awareness at creation time which cannot be changed afterwards. Only newly created elements will be affected

**Comment by HeikoKlare** [comment]
I agree that this is probably not possible (or rather difficult), as also indicated by recent finding:
- https://github.com/vi-eclipse/Eclipse-Platform/issues/181

Even if it was possible, it is quenstionable whether it is worth the effort. Changes to this setting should be very seldom (and hopefully in the long term not necessary at all), so I would be in favor of accepting that for such a change the application simply has to be restarted.

Thus closing this issue as not planned.
